docs/user_guide/adding_trials.rst

```diff
@@ -0,0 +1,30 @@
+.. _adding_trials:
+
+Adding Trials to NWB Files
+==========================
+
+NWB allows you to store information about time intervals in a structured way. These structure are often used to store
+information about trials, epochs, or other time intervals in the data.
+You can add time intervals to an NWBFile object before writing it using PyNWB.
```

Comment:
```suggestion

```

Not sure what this sentence adds since there is no import or explicit usage from the `pynwb` module in this example (only utilizing built-in methods of the `NWBFile` class; which yes, comes from `pynwb` of course, but the constructing of the neurodata object occurs implicitly)